core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -596,4 +632,223 @@ public static RunLengthEncodedBlock createRLEBlock(long value, int positionCount
         BIGINT.writeLong(blockBuilder, value);
         return new RunLengthEncodedBlock(blockBuilder.build(), positionCount);
     }
+
+    public static DictionaryBlock createRandomDictionaryBlock(Block dictionary, int positionCount)
+    {
+        checkArgument(dictionary.getPositionCount() > 0, format("dictionary position count %d is less than or equal to 0", dictionary.getPositionCount()));
+
+        int[] ids = IntStream.range(0, positionCount)
+                .map(i -> RANDOM.nextInt(dictionary.getPositionCount()))
+                .toArray();
+        return new DictionaryBlock(0, positionCount, dictionary, ids, false, randomDictionaryId());
+    }
+
+    public static Block createRandomBlockForType(
```

Comment:
could we have a simpler version of this method, e.g:
`createRandomBlockForType(Type type, int positionCount, double nullRate)`
?